Sources/Networking/ImageDownloader.swift

```diff
@@ -43,6 +43,17 @@ public struct ImageLoadingResult {
 
     /// The raw data received from downloader.
     public let originalData: Data
+
+    /// Creates an `ImageDownloadResult`
+    ///
+    /// - parameter image: Image of the download result
+    /// - parameter url: URL from where the image was downloaded from
+    /// - parameter originalData: The image's binary data
+    public init(image: KFCrossPlatformImage, url: URL? = nil, originalData: Data) {
```

Comment:
I let Xcode generate the `init` and that's what it produced. Probably since it matches the property type. But if you would prefer having it non-optional, I see no reason why not to change it.

Reply:
Ummm. Interesting.

OK, since the generator prefers an optional one, let's just follow it.

Thanks!